### ISSUE TITLE
Zoom in/out causes GUI freezes

### ISSUE THREAD

**Issue by Felixoid** [wez/wezterm#355]
## Describe the bug

Freshly installed wezterm faces freezes on CTRL-+ / CTRL-- keystrokes

## Environment

 - OS: arch linux (btw, for sure)
 - Version: wezterm 20201101-103216-403d002d-70-gc7dc44a5

## To Reproduce

The way software is [build](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wezterm-git&id=884c16364352edcdffada76d863cb65c5c790f7f#n27) is just:

```
git clone https://github.com/wez/wezterm.git && cd wezterm
git submodule update --init --recursive
cargo build --release
```

Then I run `wezterm`, press ^+ or ^-, and the GUI freezes.

## Configuration

There are no config files used.

## Expected behavior
The terminal continues to work

## Screenshots

<details><summary>session record</summary><p>

![output](https://user-images.githubusercontent.com/3025537/100377346-6448c500-3011-11eb-8104-d06cd2d09300.gif)

</details>

## Logs

```
$ wezterm
 2020-11-26T16:47:20.697Z ERROR wezterm_gui::gui::termwindow > OpenGL initialized! Mesa Intel(R) HD Graphics 620 (KBL GT2) OpenGL ES 3.2 Mesa 20.2.2 is_context_loss_possible=false
 2020-11-26T16:47:29.555Z ERROR wezterm_gui::gui::termwindow > scale changed so resize to RowsAndCols { rows: 24, cols: 80 } Dimensions { pixel_width: 720, pixel_height: 475, dpi: 96 }
 2020-11-26T16:47:29.677Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:31.374Z ERROR wezterm_gui::gui::termwindow > scale changed so resize to RowsAndCols { rows: 24, cols: 80 } Dimensions { pixel_width: 640, pixel_height: 425, dpi: 96 }
 2020-11-26T16:47:31.504Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:32.086Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:32.447Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:32.682Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:32.811Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:32.961Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:33.668Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:34.016Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:34.296Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:35.852Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:36.303Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:36.351Z ERROR wezterm_term::terminalstate  > Ding! (this is the bell)
 2020-11-26T16:47:37.321Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:37.369Z ERROR wezterm_term::terminalstate  > Ding! (this is the bell)
 2020-11-26T16:47:37.926Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:37.996Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:38.167Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:38.420Z ERROR wezterm_gui::gui::termwindow > paint_opengl_pass failed: FreeType error 6 0x6
 2020-11-26T16:47:38.512Z ERROR mux                          > read_pty EOF: pane_id 0
 2020-11-26T16:47:38.515Z ERROR mux::localpane               > Pane id 0 is_dead
 2020-11-26T16:47:38.515Z ERROR mux                          > tab 0 is dead
```

**Comment by wez** [comment]
Fixed in 164adb78e30c6d5f36ef4e5d3141e07dd5779a29

**Comment by Felixoid** [comment]
Thank you, the problem is gone, indeed!
